scripts/umccr_pipeline/UMCCR_Library_Tracking_MetaData.js

```diff
@@ -2,7 +2,7 @@
 // Should be attached to the UMCCR_Library_Tracking_MetaData Google Spreadsheet
 
 var CURRENT_YEAR = '2022'
```

Comment:
`CURRENT_YEAR` has no longer usage within the script scope. We could remove it, perhaps.